examples/templates/js/dist/src.78130f5d.css

```diff
@@ -0,0 +1,57 @@
+@charset "UTF-8";
```

Comment:
`/dist` should not be committed to the repo

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/b694ead6d2cc10c5a0f884aa083be0b15e0adebf